src/Core/Domain/ValueObject/ConfigurableCommand.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Domain\ValueObject;
+
+use InvalidArgumentException;
+use SprykerSdk\SdkContracts\Entity\CommandInterface;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+use SprykerSdk\SdkContracts\Entity\ConverterInterface;
+use SprykerSdk\SdkContracts\Entity\ExecutableCommandInterface;
+
+class ConfigurableCommand implements CommandInterface, ExecutableCommandInterface
+{
+    /**
+     * @var \SprykerSdk\SdkContracts\Entity\CommandInterface&\SprykerSdk\SdkContracts\Entity\ExecutableCommandInterface
+     */
+    protected CommandInterface $command;
+
+    /**
+     * @var bool|null
+     */
+    protected ?bool $stopOnError;
+
+    /**
+     * @var array|null
+     */
+    protected ?array $tags;
```

Comment:
The same as for $stopOnError, lets use `array` with default value `[]`

Reply:
The same